src/openMVG/robust_estimation/rand_sampling.hpp

```diff
@@ -35,10 +35,11 @@
 namespace openMVG {
 namespace robust{
 
+// rperrot : is this legal ? why using an anonymous namespace here ? 
 namespace
 {
   std::default_random_engine random_generator;
```

Comment:
In order to do not declare it as static. I added the thread_local close too in the develop branch